content/participants/ttjc@Uyadav207/index.md

```diff
@@ -25,73 +30,120 @@ I want mentorship.
 
 ## something interesting about you?
 
+Quench my programming thirst by writing Tech Blogs.
 I Love to play table tennis.
-I use to carry Tech-savvy kind of personality.
+Github contributions at night.
 I love talking to people, with different opinions and share some knowledge.
 
 ### help I need?
 
-It would be good if someone tells me how to prepare for the job interview and project management.
+It would be good if someone tells me how to prepare for the job interview, Project management and how to make good professional relations. 
 
 ### how to contact me?
 
 - [Email](mailto:yadavutkarsh207@gmail.com)
 - [LinkedIn](https://www.linkedin.com/in/utkarsh-yadav-37242b17b/)
-- [Twitter](https://twitter.com/UTKARSH40528132/)
+- [Twitter](https://twitter.com/utkarsh_js_dev)
+- [Instagram](https://www.instagram.com/utkarsh_developer/)
+- [Portfolio](https://utkarshdeveloper.netlify.app/)
+- [DEV.to](https://dev.to/uyadav207)
 
 ### projects
+
+Some of my latest Projects are below.
+
+> Can visit [Here](https://utkarshdeveloper.netlify.app/) for my all projects and some UI designs.
+
 My projects:
 
-#### **TubeStore**: A YouTube Clone Application.
+#### 1. **Relaxify** : A React Music Player ⚛
 
-_description_ : User can search for over 10 Billion Videos, can play and Enjoy, Learn, Entertain themselves.
+_description_ : Relaxify is a Spotify music player. Build using React | Node.js | Spotify Playback SDK that allows the user to connect to any Spotify device and play the music on web.
+
+_stack_: `React.Js` | `Spotify's PlayBack SDK` | `Spotify's web API` | `Node.Js`
+
+_hosted link_: https://relaxify.netlify.app/
 
-_stack_: React-Native | Redux | Google API | Expo-CLI.
+_github link_: https://github.com/Uyadav207/relaxify
 
-_hosted link_: Will Post Soon
+#### 2. **TubeStore** : A YouTube Clone Application 📱
```

Comment:
Hey! @Uyadav207 have you hosted these react-native applications on play store? If you have you should add a link to that here.

Reply:
@khusharth, Not yet.
But will Update it Sooner.

Reply:
Ok then :+1: